providers/factories/oracle/marketmap.go

```diff
@@ -33,49 +34,21 @@ func DefaultMarketMapProvider(
 		Transport: &http.Transport{MaxConnsPerHost: cfg.API.MaxQueries},
 		Timeout:   cfg.API.Timeout,
 	}
-	requestHandler, err := apihandlers.NewRequestHandlerImpl(client)
-	if err != nil {
-		return nil, err
-	}
+	var apiDataHandler types.MarketMapAPIDataHandler
+	var requestHandler apihandlers.RequestHandler
+	var ids []types.Chain
 
-	queryHandler, err := types.NewMarketMapAPIQueryHandler(
-		logger,
-		cfg.API,
-		requestHandler,
-		apiDataHandler,
-		apiMetrics,
-	)
-	if err != nil {
-		return nil, err
+	switch cfg.Name {
+	case dydx.Name:
+		apiDataHandler, err = dydx.NewAPIHandler(cfg.API)
+		ids = []types.Chain{{ChainID: dydx.ChainID}}
+	default:
+		return nil, fmt.Errorf("unknown provider: %s", cfg.Name)
```

Comment:
This case should return the default marketmap API handler that hits `x/marketemap` right?

or is the idea that we'll fill this info out later after the refactor?